less/viewdetails.less

```diff
@@ -524,22 +525,21 @@
   }
 } /*.view-details-section*/
 
-
 .view-details-map {
+  height: 230px!important;
+  margin-bottom: 30px;
+  display: block;
+  background-color: rgba(164, 164, 166, 0.24);
+  
+  @media @tablet {
+    height: 200px !important;
+  }
+  @media @phone {
+    height: 150px !important;
+    margin-bottom: 0;
+  }
   .map {
-    width: 100%;
-    height: 280px !important;
-    margin-bottom: 30px;
-
-    @media @tablet {
-      height: 220px !important;
-    }
-    @media @phone {
-      height: 170px !important;
-    }
-    @media @phone {
-      margin-bottom: 0;
-    }
+    height: 100%;
   }
   /*is 100% by defaul - it stretches the map!*/
   .ol-unselectable {
```

Comment:
If you could get rid of this style on ` .ol-unselectable`, https://github.com/c2corg/v6_ui/issues/612 could be fixed as well.